### DISCUSSION TITLE
Space pool and NFT

### DISCUSSION THREAD

**Discussion by jcpamart** [madMAx43v3r/chia-plotter#736] [Q&A]
Hi,
Madmax seems great, I have tried many pool and solo, and all is fine.
But with space pool, it run under the CHIA 1.1.795, with NFT protocol.

1-Does the last release of Madmax can plot the nft plot ?
2-Can I convert my old plot to nft compatible ?

Thanks for your help
Best regards
JC

**Comment by Dartellum** [comment]
Did you see the instructions provided by Space Pool? You need to use the pool-puzzles branch. See https://blog.pool.space/get-an-early-start-with-space-pool-mainnet-8855348826a0. I do not know if you are using Windows or Linux but the Linux instructions provide details on how to compile the pool-puzzle branch.

**Reply by rattfieldnz** [reply]
I followed the same link as @Dartellum and plotting the NFT plots with MadMax works. Instead of using the -p parameter for your pool public key, use '-c' followed by your NFT pool contract address. Here is the script I am using below to do this (an adaptation of the .cmd file already with MadMax). Take note of the line that begins with 'powershell'):

```
@ECHO OFF

SET hr=%time:~0,2%
IF "%hr:~0,1%" equ " " set hr=0%hr:~1,1%
SET DATETIME=Log_%date:~-4,4%%date:~-10,2%%date:~-7,2%_%hr%%time:~3,2%%time:~6,2%
SET LOG_FILE=logs/%DATETIME%.log

IF NOT EXIST logs MKDIR logs
IF NOT EXIST D:\ChiaTemp\ MKDIR D:\ChiaTemp\
IF NOT EXIST D:\ChiaTemp\MadMax\ MKDIR D:\ChiaTemp\MadMax\
IF NOT EXIST D:\ChiaTemp\MadMax\PoolPlots\ MKDIR D:\ChiaTemp\MadMax\PoolPlots\

SET PLOT_TEMP_DIR=D:\ChiaTemp\MadMax\PoolPlots\
SET PLOT_DIR=E:\ChiaPoolPlots\
SET POOL_CONTRACT_KEY=yourpoolcontractkey
SET FARMER_KEY=yourfarmerkey

rem Below two lines delete any plot tmp files, in case of previous plotter crash
rmdir /s /q %PLOT_TEMP_DIR%
mkdir %PLOT_TEMP_DIR% 

powershell ".\chia_plot.exe -n 5 -r 16 -u 256 -v 256 -t %PLOT_TEMP_DIR% -d %PLOT_DIR% -c %POOL_CONTRACT_KEY% -f %FARMER_KEY% | tee '%LOG_FILE%'"

PAUSE
```

If you would rather watch a video tutorial (as I did to double check things), you can watch this VidTut here: https://www.youtube.com/watch?v=UY7e3f-F4L8.

**Reply by rattfieldnz** [reply]
As a side note, I have been using the MadMax plotter executable with [PSChiaPlotter](https://github.com/MrPig91/PSChiaPlotter), which provides a GUI interface for entering fields like pool contract keys.

**Reply by alt-savage** [reply]
Old plot (called OG plots) CANNOT be converted to new portable pool plots (NFT plots).  Just farm them both, and replot over time to replace the old plots with new plots. It's a bummer to have to delete old plot to make way for the new plots, but hopefully only need to replot once.
